docs/js/set-return-focus-function.js

```diff
@@ -0,0 +1,41 @@
+const { createFocusTrap } = require('../../dist/focus-trap');
+
+const container = document.getElementById('setreturnfocus-function');
+let clickedElement;
+
+const setReturnFocus = ({ previousActiveElement }) => {
+  if (clickedElement && clickedElement.id === 'focus-this') {
+    return clickedElement;
+  } else if (clickedElement && clickedElement.id === 'focus-initial') {
+    return previousActiveElement;
+  }
+  return false;
+};
+
+const focusTrap = createFocusTrap('#setreturnfocus-function', {
+  onActivate: () => container.classList.add('is-active'),
+  onDeactivate: () => container.classList.remove('is-active'),
+  setReturnFocus,
+  allowOutsideClick: true,
```

Comment:
It would be nice to use the function here to limit acceptable clicks to those that are on the deactivate buttons. Otherwise, I can go activate another trap, and that may seem odd to some:

![image](https://user-images.githubusercontent.com/2855350/132421214-1b6125b6-d0fa-45c8-9b22-ebff540ba95d.png)